suite-native/app/e2e/pageObjects/accountDetailActions.ts

```diff
@@ -1,11 +1,21 @@
 import { expect as detoxExpect } from 'detox';
 
 class AccountDetailActions {
+    async waitForScreen() {
+        await waitFor(element(by.id('@screen/AccountDetail')))
+            .toBeVisible()
+            .withTimeout(5000);
```

Comment:
The timeout values throughout this PR are chosen based on experience? I think it may be a good idea to just define a few different constants globally and just go with them, instead of picking a new number everytime.

Reply:
The values are empirically tested, so the test is as less flaky as possible. But I agree that introducing some predefined values would make sense. Since this is not a problem only for the PR but for the whole e2e codebase, let's revisit this issue with some follow-up PR. Do you have any recommendations on how high the individual values should be? I will be happy to discuss that.